.github/workflows/solo-e2e-test.yml

```diff
@@ -260,4 +260,40 @@ jobs:
           echo "Block is $BLOCK"
           echo "BLOCK=$BLOCK" >> $GITHUB_ENV
 
-      # TODO verify MN using curl and REST API
+      - name: Get firstBlock
+        run: |
+          FIRSTBLOCK=$(grpcurl \
+            -plaintext \
+            -import-path protobuf-sources/src/main/proto \
+            -import-path protobuf-sources/block-node-protobuf \
+            -proto block-node/api/block_access_service.proto \
+            -d '{"block_number": "0"}' \
```

Comment:
I would get the first block based on the earliest block from serverStatus.
In the previous section we should make it get earliest and latest.
The getBlock on the BN and MN can use those values.
Start with the earliest for MN though. Depending on latency we'll want to be smart on when to query for latest on MN, it should work but something to watch out for and potentially separate in case you see issue initially